modules/json/src/helpers/execute-function.js

```diff
@@ -0,0 +1,17 @@
+// This attempts to execute a function
+export function executeFunction(targetFunction, props, configuration) {
+  // Find the function
+  const Function = configuration.functions[targetFunction];
```

Comment:
Nitpick, not a fan of `Function` as a variable name, I'd stick with `matchedFunction`